content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -20,6 +20,20 @@ The annotations apply to audit events. Audit events are different from objects i
 
 <!-- body -->
 
+## k8s.io/deprecated
+
+Example: `k8s.io/deprecated: "true"`
+
+Value **must** be boolean. It is set to "true" indicating the request used a deprecated API version.
```

Comment:
```suggestion
Value **must** be "true" or "false". The value "true" indicates that the request used a deprecated API version.
```

Reply:
That looks good. I'll make the changed later today. Just waiting in case more reviews are made.